### PR TITLE
Modal alert

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: periscope2
 Type: Package
 Title: Enterprise Streamlined 'shiny' Application Framework Using 'bs4Dash'
-Version: 0.1.3
+Version: 0.1.3.9000
```
Comment (cb4ds) [inline DESCRIPTION:4]: The version should be 0.1.4.9000, this is the development (.9000) version of the next package

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
-# Revisions and Change Log
+## 0.1.3
```
Comment (cb4ds) [inline NEWS.md:1]: You need a line for 0.1.4, you already have 0.1.3 released, so there should be an 0.1.3 and an 0.1.4 line, every release needs a news line.

**Review by cb4ds — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 2 above)
Comment (agenius-mohammed-ali) [reply NEWS.md]: done

File: DESCRIPTION (hunk 1 above)
Comment (agenius-mohammed-ali) [reply DESCRIPTION]: done

**Review by cb4ds — APPROVED** [review]

**Pull request merged**